EIPS/eip-1363.md

```diff
@@ -191,11 +191,13 @@ interface ERC1363Spender {
 }
 ``` 
 
+Note that `transferAndCall`, `transferFromAndCall` and `approveAndCall` MUST revert if called on an EOA as it can't return the [ERC-1363](./eip-1363.md) expected value.
```

Comment:
```suggestion
Note that `transferAndCall`, `transferFromAndCall` and `approveAndCall` MUST revert if called on an EOA as it can't return the [EIP-1363](./eip-1363.md) expected value.
```